tcms/testcases/tests/test_views.py

```diff
@@ -64,6 +65,36 @@ def test_user_in_default_group_sees_comments(self):
                 html=False
             )
 
+    def test_user_sees_bugs(self):
+        bug_1 = BugFactory(bug_id = 1)
+        bug_2 = BugFactory(bug_id = 2)
+
+        self.case_run_1.add_bug(bug_1.bug_id, bug_1.bug_system.pk)
+        self.case_run_1.add_bug(bug_2.bug_id, bug_2.bug_system.pk)
+
+        url = reverse('caserun-detail-pane', args=[self.case_run_1.case_id])
+        response = self.client.get(
+            url,
+            {
+                'case_run_id': self.case_run_1.pk,
+                'case_text_version': self.case_run_1.case.latest_text_version()
+            }
+        )
+
+        self.assertEqual(HTTPStatus.OK, response.status_code)
+
+        print(response.content)
+
+        self.assertContains(
+            response,
+            bug_1.get_full_url(),
+            html=True)
```

Comment:
Remove `html=True` and the assert will work. Check the documentation to learn what it does.